### ISSUE TITLE
pmtiles defaults lead to bitty network when zoomed out

### ISSUE THREAD

**Issue by Robinlovelace** [Urban-Analytics-Technology-Platform/od2net#52]
There are ways to fix this by changing the arguments sent to `tippecanoe`, sure others have noticed it but cannot see an issue open on it.

Related issue is #37, I think fixing this one would yield a bigger benefit, especially for use cases where strategic planning is the aim.

![image](https://github.com/Urban-Analytics-Technology-Platform/od2net/assets/1825120/6a7fa65b-ff9b-4fa1-b2bd-477a668bf84d)

Source: following instructions in #2

**Comment by dabreegster** [comment]
Thanks for bringing this up, known issue! It's one reason I started https://github.com/Urban-Analytics-Technology-Platform/lines2pmtiles/ -- to have full control over picking features at low zooms. We want to emphasize the most important roads, aka, need something like https://github.com/felt/tippecanoe/issues/132

**Comment by Robinlovelace** [comment]
Yeah, I've hit issues when generating networks for CRUSE. Interestingly www.npt.scot doesn't seem to suffer from this issue, @mem48 may have a tip or 2. I like the idea of `lines2pmtiles`, attribute-based prioritisation is a possibility..
